react-app/src/components/first-element.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export class SecondHeader extends React.Component {
+  render() {
+    return <h2>Hello world!</h2>;
+  }
+}
+
```

Comment:
You'd be much better off following the approach "single (reusable) component, solving single problem, placed within its own file". So, I would recommend to pass all the custom messages as a properties (for better flexibility and re-usability) and place each component into its own file.